test/helpers/scheme.go

```diff
@@ -13,13 +13,13 @@ import (
 	_ "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta1/dynakube" //nolint:staticcheck
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2"
 	_ "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube" //nolint:staticcheck
-	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3"
-	_ "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta4"
+	_ "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta4/dynakube"
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 )
 
 func SetScheme(ctx context.Context, envConfig *envconf.Config) (context.Context, error) {
-	err := v1beta3.AddToScheme(envConfig.Client().Resources().GetScheme())
+	err := v1beta4.AddToScheme(envConfig.Client().Resources().GetScheme())
```

Comment:
v1beta4 should be added to Scheme collection. It shouldn't replace v1beta3.

Reply:
[5295e0b](https://github.com/Dynatrace/dynatrace-operator/pull/4529/commits/5295e0bdd6f665b5f712ac81d6432791a0a04a8c)